packages/shared/models/attachement.js

```diff
@@ -0,0 +1,11 @@
+import mongoose from 'mongoose'
+
+const attachmentSchema = new mongoose.Schema({
+  id: String,
+  filename: String,
+  addedDate: Date,
+  content: String
```

Comment:
change to:
fileHash: String
it will store Muller hash name:
https://www.npmjs.com/package/multer